### DISCUSSION TITLE
Download whole audiobook

### DISCUSSION THREAD

**Discussion by Avenger2252** [advplyr/audiobookshelf#775] [Q&A] [answered]
Hi all,
is there a way to download the whole audiobook instead of the single files from the "Library Files Section" via the web browser?
If I remember correctly that in an earlier version, it was possible to download the whole audiobook as one zip-file including all single audio files like in the app.
Thanks a lot for your help.

**Comment by advplyr** [comment, marked as answer]
There is no way right now but there is a feature request for it
